arangod/IResearch/IResearchFeature.cpp

```diff
@@ -499,6 +499,8 @@ arangodb::iresearch::IResearchFeature::WalFlushCallback registerRecoveryMarkerSu
     builder.add(FLUSH_VALUE_FIELD, value);
     builder.close();
 
+    LOG_TOPIC("hunde", INFO, arangodb::Logger::FIXME) << "walFlushCallback(): commit " << builder.slice().toJson();
```

Comment:
intentional?

Reply:
Yes, in the sense that this is still WIP and we use this for debugging during the performance tests.

Reply:
Must of course be removed before this is ready.